tls/s2n_cipher_preferences.c

```diff
@@ -358,6 +358,54 @@ const struct s2n_cipher_preferences elb_security_policy_tls_1_1_2017_01 = {
     .minimum_protocol_version = S2N_TLS11
 };
 
+struct s2n_cipher_suite *cipher_suites_elb_security_policy_tls_1_2_ext_2018_06[] = {
+    /* &s2n_ecdhe_ecdsa_with_aes_128_gcm_sha256, */
```

Comment:
I think we can uncomment these ECDSA suites now. Previously we commented it out because s2n did not yet have support for ECSDA.